scripts/build-rss.ts

```diff
@@ -0,0 +1,111 @@
+import fs from 'fs/promises';
+import json2xml from 'jgexml/json2xml';
+
+import posts from '../config/posts.json' assert { type: 'json' };
```

Comment:
Don't import posts directly here. This file is generated at the build time of the website.